package.json

```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
       "pkginfo": "~0.3.0",
-      "underscore": "~1.5.2"
+      "underscore": "1.8.3"
```

Comment:
Then this should go to the `devDependencies` so that users that depend on this lib don't have to download underscore :)